docs/terms/data_assistant.md

```diff
@@ -0,0 +1,81 @@
+
+Data Assistants really show their worth when you provide them a Batch Request that contains multiple Batches.  With previous Profilers you would only be able to introspect a single Batch at a time.  This meant that the Expectation Suite generated would only reflect a single Batch.  If you had many Batches of data that you wanted to build inter-related Expectations for, you would have needed to run each Batch individually and then manually compare and update the Expectation parameters that were generated.  With a Data Assistant, that process is automated.  You can provide a Data Assistant multiple Batches and get back Expectations that have parameters based on, for instance, the mean or median value of a column on a per-Batch basis. 
+
+### Visual plots for Metrics
+
+ When working in a Jupyter Notebook you can use the `plot_metrics()` method of a Data Assistant's result object to generate a visual representation of your Expectations, the values that were assigned to their parameters, and the Metrics that informed those values.  This assists in exploratory data analysis and fine-tuning your Expectations, while providing complete transparency into the information used by the Data Assistant to build your Expectations.
+
+## API basics
+
+Data Assistants can be easily accessed from your Data Context.  In a Jupyter Notebook, you can enter `context.assistants.` and use code completion to select the Data Assistant you wish to use.  All Data Assistants have a `run(...)` method that takes in a Batch Request and numerous optional parameters, the results of which can be loaded into an Expectation Suite for future use.
+
+The Onboarding Data Assistant is an ideal starting point for working with Data Assistants.  It can be accessed from `context.assistants.onboarding`, or from the <TechnicalTag tag="cli" text="CLI" /> command `great_expectations suite new --profile`.
+
+:::note For more information on the Onboarding Data Assistant, see the guide:
+- [How to create an Expectation Suite with the Onboarding Data Assistant](../guides/expectations/data_assistants/how_to_create_an_expectation_suite_with_the_onboarding_data_assistant.md)
+:::
+
+### Configuration
+
+Data Assistants come pre-configured!  All you need to provide is a Batch Request, and some optional parameters in the Data Assistant's `run(...)` method.
+
+## More details
+
+### Design motivation
+
+Data Assistants were designed to make creating Expectations easier for users of Great Expectations.  A Data Assistant will help solve the problem of "where to start" when working with a large, new, or complex dataset by greedily asking questions and building a list of all the relevant Expectations that it can determine from the answers to those questions.  Branching question paths ensure that additional relevant Expectations are built on the groundwork of the earlier questions asked.  The result is a comprehensive Expectation Suite that you can then edit and trim down to meet your desired use cases.
```

Comment:
@Rachel-Reverie Strictly speaking, `Expectations` is not the only output (e.g., Metrics as well, or could be only Metrics; but not sure if/how we would want to communicate this fact).  Thanks!

Reply:
Perhaps: "The result is a comprehensive gathering of Metrics that can be saved, reviewed, or used by the Data Assistant to generate a set of proposed Expectations." ?